wp-content/themes/citylimits/homepages/assets/less/homepage.less

```diff
@@ -215,7 +215,7 @@
         padding-right: 0;
         h2 {
           padding: 0;
-          font-size: 1.75rem;
+          font-size: 1.1rem;
```

Comment:
Do we want to make this same font-size change up on line 179, for tabletish screen sizes?

Reply:
@benlk The original request was
> Reduce size of top headline on mobile homepage

but I'll update it here, too, just in case

Reply:
Updated in https://github.com/INN/umbrella-citylimits/pull/136/commits/d75164db83e8a7e9cc78f451de81add7a23481f1